src/main/cli/settings.c

```diff
@@ -1598,6 +1598,7 @@ const clivalue_t valueTable[] = {
 
 // PG_POSITION
     { "position_alt_source",           VAR_INT8   | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_POSITION_ALT_SOURCE }, PG_POSITION, offsetof(positionConfig_t, altSource) },
+    { "race_mode", VAR_UINT8  | PROFILE_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_PID_PROFILE, offsetof(pidProfile_t, race_mode) },
```

Comment:
Please align the parameters with the entry above as much as possible.

Also "race_mode" is too generic a term. Maybe `angle_race_mode`.

Reply:
Done